include/deal.II/fe/fe_point_evaluation.h

```diff
@@ -48,7 +48,7 @@ namespace internal
       using gradient_type = Tensor<1, n_components, Tensor<1, dim>>;
 
       static void
-      read_value(const double &     vector_entry,
+      read_value(const double       vector_entry,
```

Comment:
I don't understand what is the motivation to switch from const-reference to plain const here. Can you explain @bangerth?

Reply:
For built-in types passing by reference/pointer often is more expensive than passing by copying. We try to be consistent here.

Reply:
Correct. Passing by value makes it easy for the compiler to reason about what is happening to a variable. We do this everywhere where the argument is just a scalar/small object.